testcafe/tests/pages/photos/photos-timeline-model.js

```diff
@@ -68,18 +72,18 @@ export default class Timeline extends Commons {
       selectors.btnModalSecondButton,
       'Modal delete button Delete'
     )
+    if (t.fixtureCtx.isVR)
+      //dates show up here, so there is a mask for screenshots
+      await t.fixtureCtx.vr.takeScreenshotAndUpload({
+        screenshotPath: screenshotPath,
+        withMask: withMask
+      })
```

Comment:
J'imagine que c'est quelque chose qui est fait souvent. On pourrait avoir une fonction qui prend le contexte, le path et le masque (ou peut-être même plutôt `takeScreenshotIfVR(fixtureCtx, screenshotOptions)`?) et qui fait ce que font les lignes 74-80

Reply:
most of the time, we don't need to check `    if (t.fixtureCtx.isVR)`
It's use only for method used in old test (without screenshots) and new ones.